src/main/java/Command/Command.java

```diff
@@ -0,0 +1,18 @@
+package Command;
+
+import Storage.Storage;
+import TaskList.TaskList;
+import Ui.Ui;
+
+import java.io.IOException;
+
+public abstract class Command {
+
+    protected boolean isExit = false;
+
+    public Command() {}
+
+    public abstract void execute(TaskList t, Ui ui, Storage storage) throws IOException;
```

Comment:
One suggestion is to allow this method to be overloaded, since not every command will require all 3 parameters to be used.